### ISSUE TITLE
Missing kubetctl

### ISSUE THREAD

**Issue by analytik** [pires/kubernetes-vagrant-coreos-cluster#67]
Hi,

I'm probably doing something terribly wrong, but I'm using Windows as host with the latest Vagrant, VirtualBox, etc.

I run vagrant up, connect to master node, and cannot find kubectl nor kubectl.sh anywhere, nor the "cluster" directory. So far the only thing I found was 
```core@master / $ ls /opt/bin/```
```kube-apiserver  kube-controller-manager  kube-register  kube-scheduler  setup-network-environment  wupiao```

Should I try to install k8s again, or am I missing something else?

**Comment by pires** [comment]
Hey @analytik, didn't you see anything like this in the terminal?
```
 "Unknown or unsupported platform: windows"
```

**Comment by analytik** [comment]
Hi @pires, thanks, but I have not seen anything like that. I only get
```
==> master: Waiting for Kubernetes master to become ready...
'until' is not recognized as an internal or external command,
operable program or batch file.
==> master: Configuring Kubernetes cluster DNS...
```
I understand if Kubernetes does not support Windows as a platform to run on, whether as a master or a minion, but why would it be a problem for Windows to run a **Vagrant** machine with Linux?

I know this a bit ridiculous, but could I get it running (as a development machine) as
```
Windows on bare metal
  - Linux in a VM
    - Vagrant inside Linux VM
       - CoreOS Master
       - CoreOS Minions
```
or do you think would it be more reliable to pick another approach completely, i.e. try to install Kubernetes directly in a Linux VM and create both a master and a minion there? I was hoping to set up a development environment from my development machine, and get some experience with managing a cluster, although that's secondary after having a working dev environment.

**Comment by pires** [comment]
The scripts I provide do not work properly in Windows systems. I'm sorry.

**Comment by analytik** [comment]
Ah. I didn't examine the vagrant setup files in detail, so I was hoping it wouldn't need anything from the host OS. My bad.

Still, thank you for investing time into this project. :)

**Comment by pires** [comment]
@analytik sorry dude. Thanks for your kind words, nonetheles.

**Comment by pires** [comment]
Reopening for proper handling of Windows hosts, namely terminate `vagrant up` with the information that Windows is not supported.

**Comment by pires** [comment]
So @Hikariii is helping on this!

**Comment by pires** [comment]
Fixed by #81.
